### ISSUE TITLE
Compilation fails

### ISSUE THREAD

**Issue by jchodera** [FoldingAtHome/fah-viewer#2]
```
g++ -o build/fah/viewer/io/XYZWriter.o -c -std=gnu++98 -ggdb -Wall -Werror -DDEBUG -D_REENTRANT -DHAVE_EXPAT -DHAVE_PTHREADS -DHAVE_LIBSQLITE -DHAVE_OPENSSL -DDEBUG_LEVEL=1 -DHAVE_CBANG -DHAVE_LIBNSL -DHAVE_LIBM -DHAVE_LIBPTHREAD -DGMX_DOUBLE -DHAVE_GROMACS -DGLEW_STATIC -DUSING_CBANG -I/cbang/src -I/cbang/include -I/boost_1_55_0 -I/openssl-1.0.2d/include -I/usr/include/freetype2 -I/root/fah-gromacs -Ibuild build/fah/viewer/io/XYZWriter.cpp
In file included from build/fah/viewer/io/XYZWriter.cpp:29:
build/fah/viewer/io/XYZWriter.h: In constructor 'FAH::XYZWriter::XYZWriter(const cb::OutputSink&)':
build/fah/viewer/io/XYZWriter.h:44: error: no matching function for call to 'cb::OutputSink::OutputSink(void (&)()throw ())'
/cbang/src/cbang/io/OutputSink.h:52: note: candidates are: cb::OutputSink::OutputSink(std::ostream&, const std::string&)
/cbang/src/cbang/io/OutputSink.h:51: note:                 cb::OutputSink::OutputSink(const std::string&)
/cbang/src/cbang/io/OutputSink.h:49: note:                 cb::OutputSink::OutputSink(char*, std::streamsize, const std::string&)
/cbang/src/cbang/io/OutputSink.h:44: note:                 cb::OutputSink::OutputSink(const cb::OutputSink&)
scons: *** [build/fah/viewer/io/XYZWriter.o] Error 1
scons: building terminated because of errors.
```
I only tried compiling this because it is apparently needed (!!!) as a dependency for building `fah-client`.

**Comment by jchodera** [comment]
There are no release versions of anything, so I can only assume the current head versions of everything are supposed to be compiled together, but it seems like that assumption has failed the empirical test.

**Comment by jchodera** [comment]
Fixed by #3
